src/data/shops.txt

```diff
@@ -1,139 +1,139 @@
-landfillstore	COIN	The Dinsey Company Store
-lathe	COIN	Your SpinMaster&trade; lathe
-ltt	COIN	LT&T Gift Shop
-lunarlunch	COIN	Lunar Lunch-o-Mat
-madeline	NPC	Madeline's Baking Supply
-mariogear	COIN	Mushroom District Gear Shop
-marioitems	COIN	Mushroom District Item Shop
-mayoclinic	NPC	The Mayo Clinic
-meatsmith	NPC	Meatsmith's Shop
-mrreplica	COIN	Replica Mr. Store
-mrstore2002	COIN	Mr. Store 2002
-mystic	CONC	The Crackpot Mystic's Shed
-nerve	NPC	Nervewrecker's Store
-nina	COIN	Ni&ntilde;a Store
-olivers	COIN	Fancy Dan the Cocktail Man
-piraterealm	COIN	PirateRealm Fun-a-Log
-pokefam	COIN	The Pok&eacute;mporium
-rumple	CONC	Rumplestiltskin's Workshop
-sbb_jimmy	COIN	Buff Jimmy's Souvenir Shop
-september	COIN	Sept-Ember Censer
-shadowforge	CONC	The Shadow Forge
-shakeshop	COIN	Ye Newe Souvenir Shoppe
-shoeshop	COIN	Legitimate Shoe Repair, Inc.
-shore	COIN	The Shore, Inc. Gift Shop
-si_shop1	COIN	The SHAWARMA Initiative
-snackbar	NPC	Huggler Memorial Colosseum Snack Bar
-snowgarden	CONC	Winter Gardening
-spacegate	COIN	Spacegate Fabrication Facility
-spant	CONC	Spant Bit Assembly
-starchart	CONC	A Star Chart
-still	CONC	Nash Crosby's Still
-sugarsheets	CONC	Sugar Sheet Folding
-taco_dan	COIN	Taco Dan's Taco Stand
-thankshop	COIN	A traveling Thanksgiving salesman
-topiary	COIN	Topiary Nuggletcrafting
-town_giftshop.php	NPC	Gift Shop
-toxic	COIN	Toxic Chemistry
-trapper	COIN	The Trapper
-tweedle	NPC	The Tweedleporium
-twitchsoup	COIN	The Primordial Soup Kitchen
-unclep	NPC	Uncle P's Antiques
-vault1	NPC	Fallout Shelter Medical Supply
-vault2	NPC	Fallout Shelter Electronics Supply
-vault3	NPC	Underground Record Store
-voteslime	CONC	Mad Sliemce
-warbear	COIN	Warbear Black Box
-wereprofessor_tinker	CONC	Tinkering Bench
-whitecitadel	NPC	White Citadel
-wildfire	NPC	FDKOL Auxiliary
-xo	CONC	XO Shop
+2
+5dprinter	Xiblaxian 5D printer	CONC	FIVE_D
+airport	Elemental Duty Free, Inc.	CONC	DUTYFREE
+applestore	The Applecalypse Store	COIN
+arcade	Arcade Ticket Counter	COIN
+armory	Armory and Leggery	NPCCOIN
+bacon	Internet Meme Shop	COIN
+bartender	The Typical Tavern	NPC
+bartlebys	Barrrtleby's Barrrgain Books	NPC
+batman_cave	Bat-Fabricator	COIN
+batman_chemicorp	ChemiCorp	COIN
+batman_orphanage	Gotpork Orphanage	COIN
+batman_pd	Gotpork P. D.	COIN
+beergarden	Beer Garden	CONC	BEER
+blackmarket	The Black Market	NPCCOIN
+boutique	Paul's Boutique	COIN
+brogurt	The Frozen Brogurt Stand	COIN
+bugbear	Bugbear Bakery	NPC
+campfire	Your Campfire	COIN
+canteen	The Canteen	COIN
+caveshop	The Neandermall	COIN
+chateau	Chateau Mantegna Gift Shop	NPC
+chinatown	Chinatown Shops	NPC
+conmerch	KoL Con 13 Merch Table	COIN
+crimbo12	Uncle Crimbo's Futuristic Trailer	CONC	CRIMBO12
+crimbo14	Crimbo 2014	COIN
+crimbo16	Crimbo Lumps Shop	CONC	CRIMBO16
+crimbo17	Cheer-o-Vend 3000	COIN
+crimbo18	The Crimbo Cafe (2018)	NPC
+crimbo18giftomat	Crimbo Town Gift-O-Mat	NPC
+crimbo19	The Crimbo Cafe (2019)	NPC
+crimbo19toys	The H. M. S. Kringle's Workshop	KRINGLE	CONC
```

Comment:
no other lines end in CONC -- they are CONC (tab) and then the ENUM.

Reply:
Fixed.